### DISCUSSION TITLE
Bug bounty $200,-: Vulkan fullscreen

### DISCUSSION THREAD

**Discussion by Aggror** [stride3d/stride#1334] [Contributors]
Currently full screen Vulkan support is not working and we would like to get this fixed. See the official support ticket: https://github.com/stride3d/stride/issues/1024

 Open collective page: https://opencollective.com/stride3d/projects/bug-bounty-vulkan-fullscreen

**Interested**? please follow steps below:
- Reply in this topic, if you are interested in fixing this problem and tag @stride3d/stride-contributors 
- Details are discussed on how to Approach, deliverables
- Once a Stride contributor has given a go, the bug is 'reserved' on your name

Payment:
- Any PR to the Stride repo must be reviewed by a contributor and discussed if needed.
- Once a PR is reviewed and merged to the Stride repo, the submitter 60% of the bounty.
- The submitter receives the remaining 40% when a Stride release happens.

**Comment by Eideren** [comment]
For anyone investigating this issue, here's how you set and build projects under vulkan:
https://github.com/stride3d/stride/wiki/Graphics-API
Changing fullscreen state is done through `Game.Window.IsFullscreen = true;`

**Comment by cjens00** [comment]
The Open Collective page is showing that this has been resolved. Is that the case?

**Reply by Eideren** [reply]
Yes, through #1709
